workers.rb

```diff
@@ -135,20 +135,19 @@ class DOIWorker
   # Including this means we can talk to GitHub from the background worker.
   include GitHub
 
-  def perform(nwo, issue_id, config, custom_branch, clear_cache=false)
+  def perform(nwo, issue_id, config, custom_branch)
     config = OpenStruct.new(config)
     set_env(nwo, issue_id, config)
 
-    # Trying to debug a race condition on Heroku
-    sleep(10)
     # Download the paper
-    stdout, stderr, status = download(issue_id, clear_cache)
+    stdout, stderr, status = download(issue_id)
```

Comment:
Don't need to clear the cache now as each spawn of a job is guaranteed to have its own directory.